### ISSUE TITLE
Missing method

### ISSUE THREAD

**Issue by elgatito** [mattn/go-isatty#31]
After you have added separate source for android builds, gin is failing because cannot find `IsCygwinTerminal` for android build.
```
../../gin-gonic/gin/logger.go:201:67: undefined: isatty.IsCygwinTerminal
```

**Comment by sohaha** [comment]
Also encountered...

**Comment by mattn** [comment]
What OS?

**Comment by elgatito** [comment]
@mattn Android.

It should follow same function headers, as Linux does. This one is missing:
```
func IsCygwinTerminal(fd uintptr) bool 
```

**Comment by mattn** [comment]
https://github.com/mattn/go-isatty/pull/33 should fix this.

**Comment by mattn** [comment]
Closed by #33
